{"name": "test_broken", "status": "broken", "statusDetails": {"message": "Exception: oops", "trace": "def test_broken():\n>       raise Exception('oops')\nE       Exception: oops\n\ntest_alluredemo.py:20: Exception"}, "start": 1616499119927, "stop": 1616499119928, "uuid": "f76475cf-b016-4c68-a951-51a49f69d448", "historyId": "fb799eecaee64eb20ac70a0047ddd7ff", "testCaseId": "54327feac6bee8ccd45a88d670bf2886", "fullName": "test_alluredemo#test_broken", "labels": [{"name": "suite", "value": "test_alluredemo"}, {"name": "host", "value": "PC-201705071250"}, {"name": "thread", "value": "9252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_alluredemo"}]}